apps/test/unit/tutorialExplorer/backButtonTest.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../util/configuredChai';
+import BackButton from '@cdo/apps/tutorialExplorer/backButton';
+import i18n from '@cdo/tutorialExplorer/locale';
+
+describe('BackButton', () => {
+  it('renders like this', () => {
```

Comment:
Maybe this could be more descriptive? e.g. 'renders a left arrow back button`.

Reply:
:+1: Good call.  Done!